src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx

```diff
@@ -21,10 +37,28 @@ export const AUCTION_ARTWORKS_IMAGE_HEIGHT = 160
 
 export const AuctionArtworksRail: React.FC<AuctionArtworksRailProps> = ({
   sale,
+  tabType,
   ...rest
 }) => {
   const ref = useRef<HTMLDivElement | null>(null)
+  const { trackEvent } = useTracking()
   const { isEnteredView, Waypoint } = useLazyLoadComponent()
+  const { contextPageOwnerType } = useAnalyticsContext()
```

Comment:
Generally speaking, every tracking event needs this `contextPageOwnerType`, which can be pulled from our handy hook.